pkg/utils/miniojob/types.go

```diff
@@ -172,14 +172,18 @@ func (jobCommand *MinIOIntervalJobCommand) createJob(_ context.Context, _ client
 		},
 	}
 	baseEnvFrom = append(baseEnvFrom, jobCommand.CommandSpec.EnvFrom...)
+	scheme := "http://"
+	if isTLS {
+		scheme = "https://"
+	}
```

Comment:
Not sure why you include `://` into the scheme. It would make more sense doing that during `fmt.Sprintf`.